sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,16 +1,21 @@
 # Release History
-## 1.3.0-preview.1 (Unreleased)
+## 1.3.0-beta.1 (2020-09-11)
 
 ### New Features
 - Restoring Application Authentication APIs from 1.2.0-preview.6
+- Added respect of `CancellationToken` to the `MsalPublicClient` and `MsalConfidentialClient` ([#13201](https://github.com/Azure/azure-sdk-for-net/issues/13201))
```

Comment:
This should go under "Fixes and Improvements" not features, also I might reword it slightly to something like:

> Updated `MsalPublicClient` and `MsalConfidentialClient` to respect `CancellationToken` during initialization.

Reply:
Fixed